README.md

```diff
@@ -1,37 +1,34 @@
 
-Our goal is to provide the Magento community with comprehensive and quality documentation on resolving Magento issues, Magento best practices, and sharing answers to questions our Support team typically gets. We believe to accomplish that goal, we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for improving Magento Support documentation.
+Our goal is to provide the Adobe Commerce Community with comprehensive and quality documentation on resolving Magento issues, Magento best practices, and sharing answers to questions our Support team typically gets. We believe that to accomplish this goal, we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for improving Adobe Commerce Support Knowledge Base documentation.
 
-
-### Contributing 
+## Contributing 
 
 We welcome all kinds of contributions, from minor typo corrections to complete troubleshooting articles. Contributing to this repo gets you reward points, similar to contributing to Magento code and Magento Developer Documentation. See [Contribution reward points](docs/contribution-points.md) for details.
 
-
-#### General contribution flow
-
-1. Fork this repo.
-1. Make edits on forked repo.
-1. Submit a Pull Request (PR) to this repo.
-1. Tests are run:
-  * Adobe CLA - making sure the Adobe Open Source Contributor License Agreement is signed.
-  * Markdown Linting test - making sure markdown syntax is correct.
-  * File structure validation test - making sure the commit is done according to the [required file structure](.github/CONTRIBUTING,md#file_structure).
-1. PR approvals flow:
-     1. Knowledge Base (KB) writers reviews the PR within several days' time frame and adds label.
-     1. KB writer can approve/deny/request changes.
-     1. If approved, KB writer adds labels corresponding to the level of input provided in PR and internal subject matter expert (SME) reviews the PR.
-     1. SME can approve/deny/request changes.
-1. Once all corrections are done (if any requested), and both the KB writer and SME approve the PR, the KB writer imports content to the internal repo and merges it internally.
-1. The  magento/knowledge-base repo synchronizes with the internal one in 20 minutes.
-1. Once the repos are synced, your PR gets closed and you get reward points.
-
-For details on contribution flow, please refer to the [Contributor's Guide](.github/CONTRIBUTING.md).
-For templates, style guide, and formatting guidelines, please refer to [Documentation](docs/index.md).-->
-
-### Licensing
-
-This project is licensed under the OSL 3.0 License. See [LICENSE](LICENSE) for more information.
+### General Contribution Flow
+
+- Fork this Repository.
+- Make edits on the forked Repository.
+- Submit a Pull Request (PR) to this Repository.
+- Tests are run:
+  - Adobe CLA - making sure the Adobe Open Source Contributor License Agreement is signed.
```

Comment:
Linting is giving an md0007 error - if this repo is set up as devdocs then the 2nd level ordered list needs to be indented by 3 spaces not 2 - will need to be done for all 2nd level list items below

Reply:
Thank you **Barny Shergold**, I tried giving 3 spaces and also 2 spaces. still, it's throwing MD0007 issue. Can you please suggest the changes?

Reply:
Just been testing this locally - try indenting the second level by FOUR spaces. I did this and locally MDL returned no errors

Reply:
BOOM! it worked Thank you so much Barny Shergold